KerbalStuff/blueprints/api.py

```diff
@@ -476,12 +474,13 @@ def update_user_background(username: str) -> Union[Dict[str, Any], Tuple[Dict[st
     if not current_user.admin and current_user.username != username:
         return {'error': True, 'reason': 'You are not authorized to edit this user\'s background'}, 403
     user = User.query.filter(User.username == username).first()
-    base_name = secure_filename(user.username)
-    base_path = f'{base_name}-{time.time()!s}_{user.id!s}'
```

Comment:
None of the f-string tutorials I found explained what the `!s` does. I had to read the official PEP document.

Reply:
Same, I could also only find information on it in the PEP. But as far as I remember it said it was deprecated, so I didn't keep it. I'm not sure if I've ever used these flags anywhere, the implicit string conversion usually does what I want.

Reply:
Yeah, it looks like it's a no-op.